CoffeeChat/Cells/OnboardingTableViewCell.swift

```diff
@@ -8,15 +8,26 @@
 
 import UIKit
 
-class InterestsTableViewCell: UITableViewCell {
+class OnboardingTableViewCell: UITableViewCell {
 
     // MARK: Private View Vars
-    private let categoriesLabel = UILabel()
     private let cellBackgroundView = UIView()
     private let interestImageView = UIImageView()
     private let titleLabel = UILabel()
+    private lazy var categoriesLabel: UILabel = {
+        let categoriesLabel = UILabel()
+        categoriesLabel.textColor = .textLightGray
+        categoriesLabel.font = .systemFont(ofSize: 12, weight: .regular)
+        cellBackgroundView.addSubview(categoriesLabel)
+        return categoriesLabel
+    }()
```

Comment:
curious why we want to make this lazy?

Reply:
The same cell is used for the `OnboardingInterestsViewController` and the `GroupsViewController`, and the Interests show an additional label that gives examples of the interests. I made it lazy since on the Groups VC it will probably never have to use the label / constrain it onto the cell